SYCL/Basic/image/image.cpp

```diff
@@ -2,7 +2,7 @@
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
 // RUN: %HOST_RUN_PLACEHOLDER %t.out
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
-// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: env SYCL_PI_CUDA_ENABLE_IMAGE_SUPPORT=1 %GPU_RUN_PLACEHOLDER %t.out
```

Comment:
can we add this variable to %GPU_RUN_PLACEHOLDER when target BE is HIP/CUDA (in SYCL/lit.cfg.py)?
That will allow avoiding a change of multiple files and will also enable this option for all new tests.

Reply:
I considered the solution you proposed and opted for the implemented one for two reasons: enable this _experimental_ functionality just to the image tests in order to avoid possible side effects on the other ones (please consider that after https://github.com/intel/llvm/pull/5256, images will be no more enabled by default), show clearly when this experimental feature is necessary for the actual test.

Reply:
Please, let me know what you think. If needed I can also apply your suggestion.

Reply:
The current implementation affects non-CUDA BEs and may confuse developers there.
Setting the environment in the GPU placeholder for CUDA BE only looks more straightforward.

Having the environment variable set for non-image test has low risk.